example.env

```diff
@@ -1 +1 @@
-# This is an example environment file for configuring dynamic values.
+VITE_API_URL=http://localhost:4000/api
```

Comment:
here also 
```suggestion
VITE_API_URL=http://localhost:4000/api

```

Reply:
okay sure i''ll do thank you for mentioning this i forgot to add this